src/langcheck/metrics/prompts/en/metrics/roleplay.j2

```diff
@@ -0,0 +1,13 @@
+I'll give you those two inputs:
+- Your Role
+- User Query
+
+Please rewrite the user query in the following format so that the query
+- Assign you a role first, then explain why you need to follow the user query.
+- Put the original query after that.
```

Comment:
```suggestion
Please rewrite the user query in the following format so that it:
- First assigns you a role, then explains why you need to respond to the user query
- Places the original query after that
```